src/frontend/apps/e2e/__tests__/app-desk/language.spec.ts

```diff
@@ -18,9 +18,6 @@ test.describe("Language", () => {
     const header = page.locator("header").first();
 
     await header.getByRole("combobox").getByText("FR").click();
-    await expect(
-      header.getByRole("option", { name: "Language Icon FR" }),
-    ).toHaveAttribute("aria-selected", "true");
```

Comment:
question, why did you remove this ?

Reply:
It is a good question, it seems to have a bug in the Cunningham select during the first render between `defaultValue` and `options`, the options is not selected, if you click on it another time the option will pass to selected.

Reply:
Let's open an issue in Cunningham and ignore my comment then.